Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG BBKCLI_VERSION=1.0
+
+FROM alpine:3.23 AS bbkcli
+
+# See: http://www.bredbandskollen.se/bredbandskollen-cli/
+ARG BBKCLI_VERSION
+ARG TARGETPLATFORM
+RUN case ${TARGETPLATFORM} in \
+         "linux/amd64")  BBKCLI_ARCH=amd64  ;; \
+         "linux/arm64")  BBKCLI_ARCH=aarch64  ;; \
+         "linux/arm/v7") BBKCLI_ARCH=armhf  ;; \
+         "linux/386")    BBKCLI_ARCH=i386   ;; \
+    esac \
+    && echo "Fetch: https://frontend.bredbandskollen.se/download/bbk_cli_linux_${BBKCLI_ARCH:-amd64}-${BBKCLI_VERSION}" \
+    && wget -q https://frontend.bredbandskollen.se/download/bbk_cli_linux_${BBKCLI_ARCH:-amd64}-${BBKCLI_VERSION} -O /bbk_cli
```

Comment:
Uses prebuilt library. I can change it to build from source, if desired. 
Is there maybe a hash file to validate binary integrity?